.github/workflows/release.yml

```diff
@@ -21,27 +23,22 @@ jobs:
           git config user.name "github-actions"
           git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
 
-      - name: Build changelog
-        id: build_changelog
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Get previous tag
         run: |
-          export CHANGELOG=$(gh api /repos/{owner}/{repo}/releases/generate-notes -f tag_name=${GITHUB_REF#refs/tags/} --jq '.body')
-          CHANGELOG="${CHANGELOG//'%'/'%25'}"
-          CHANGELOG="${CHANGELOG//$'\n'/'%0A'}"
-          CHANGELOG="${CHANGELOG//$'\r'/'%0D'}"
-          echo "::set-output name=changelog::$CHANGELOG"
+          PREV_TAG=$(git describe --abbrev=0 --tags "${{ github.ref }}^")
+          echo "Previous release found: $PREV_TAG"
+          echo "PREV_TAG=$PREV_TAG" >> $GITHUB_ENV
```

Comment:
@devinleighsmith  @asanchezr  Do you guys know if there were changes in test that didn't make it into dev yet that this one is picking up?

Reply:
yes, this is mine.

Reply:
Correct. I made that change in test. Was going to do the back merge today so yours is picking all pending changes